lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -0,0 +1,81 @@
+.container {
+    background-color: #fff;
```

Comment:
Should use color variable `$color-bg-white` for this.

Reply:
And here.